torch/csrc/jit/passes/normalize_ops.cpp

```diff
@@ -107,6 +126,7 @@ const std::unordered_map<Symbol, Symbol>& getOperatorAliasMap() {
       {aten::true_divide_, aten::div_},
       {aten::concat, aten::cat},
       {aten::row_stack, aten::vstack},
+      {aten::rsub, aten::sub},
```

Comment:
the fix for this test
![image](https://user-images.githubusercontent.com/40742183/133736305-f8f44590-579e-4137-8149-ed3e9f0becd0.png)
seemed to be that rsub could not be found within this map, so adding it to the map and making the appropriate changes seemed to make the test pass